index.html

```diff
@@ -0,0 +1,314 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+        <link rel="stylesheet" href="styles.css">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+        <title>Cristiano Ronaldo vs Lionel Messi 2020</title>
+        <link rel="shortcut icon" type="png/image" href="/assets/images/logos/favicon.ico">
+    </head>
+
+    <body>
+        <header>
+            <nav class="firstnav">
+                <i class="fa fa-bars fa-sm"></i>
+                <img  class="youtube" src="https://www.freeiconspng.com/uploads/youtube-logo-png-transparent-image-5.png" alt="">
+            </nav>
+           
+            <form class="search">
+                <input type="text" placeholder="Search" value=""/>
+                <button><i class="fa fa-search"></i></button>
+            </form>
+
+            <img class="upload" src="/assets/images/logos/upload.png" alt="upload">
+            <img class="apps" src="/assets/images/logos/apps.png" alt="apps">
+            <img class="user" src="/assets/images/logos/user.png" alt="user"> 
```

Comment:
Images are not being displayed, you should use relative paths.
```suggestion
            <img class="upload" src="./assets/images/logos/upload.png" alt="upload">
            <img class="apps" src="./assets/images/logos/apps.png" alt="apps">
            <img class="user" src="./assets/images/logos/user.png" alt="user"> 
```

Reply:
changes effected. awaiting approval